frontend/public/components/modals/create-namespace-modal.jsx

```diff
@@ -129,20 +129,63 @@ const CreateNamespaceModalWithTranslation = connect(
         return (
           <>
             <p>{nameFormat}</p>
-            {this.props.createProject ? <p>{createNamespace}</p> : null}
+            {createProject ? <p>{createNamespace}</p> : null}
           </>
         );
       };
+
+      const popoverDescriptionText = (
+        <>
+          <p>
+            <strong>{t('public~OpenShift Projects and Kubernetes Namespaces')}</strong>
+          </p>
+          <p>
+            {t(
+              'public~Projects are exposed as editable to end users while namespaces are not. Direct creation of a project is typically restricted to administrators, while end users should use the requestproject resource.',
```

Comment:
I know this is probably the message in the design, but we should work with @megan-hall on exactly what to say here. RBAC in Kubernetes is more nuanced than this. IIRC, users can only change the display name and description of projects, not other properties (even admins).

Most users don't have access to the namespace API at all. They can't even list namespaces. Do we want to show this message to all users or only users who can access namespaces?

The API `projectrequests` rather than `requestproject`.

Reply:
Sam I am open to your feedback on what to say in the popover (particularly the nuances you mentioned above). I know it has been a while since we went over the description. 

Is it worth providing a separate project description for the users that don't have access to namespaces?